portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SearchAPI.testcase

```diff
@@ -50,20 +47,43 @@ definition {
 		var keyword = "all parameters";
 		var parameters = "size=10&displayLanguageId=en-US&page=0&groupId=${groupId}&sort=counts,desc,lastModifiedDate,desc";
 
-		task ("Go to Server Administration") {
-			ServerAdministration.openServerAdmin();
+		task ("Go to DXP and get analytics configuration") {
+			ACApi.getAnalyticsConfiguration();
 		}
 
-		task ("Get Faro Backend URL") {
-			var faroBackendURL = ACApi.getSearchHeaderValues(header = "liferayAnalyticsFaroBackendURL");
+		task ("Go to Site Page") {
+			ACUtils.navigateToSitePage(
+				pageName = "Home",
+				siteName = "guest");
 		}
 
-		task ("Get OSB-Asah-Faro-Backend-Security-Signature") {
-			var osbAsahFaroBackendSS = ACApi.getSearchHeaderValues(header = "liferayAnalyticsFaroBackendSecuritySignature");
+		task ("Type keyword in search bar") {
+			SearchPortlets.searchEmbedded(searchTerm = "${keyword}");
 		}
 
-		task ("Get OSB-Asah-Project-Id") {
-			var osbAsahProjectId = ACApi.getSearchHeaderValues(header = "liferayAnalyticsProjectId");
+		task ("Run curl command to get keyword results") {
+			ACApi.getSearchKeywords(parameters = "${parameters}");
+		}
+
+		task ("Assert response returns keyword") {
+			ACApi.assertSearchResponse(expected = "${keyword}");
+		}
+	}
+
+	@description = "Feature ID: LRAC-12058 | Automation ID: LRAC-12143 | Test Summary: Fetch keywords with displayLanguageId parameter"
+	@priority = "3"
+	test RequestKeywordsUsingDisplayLanguageId {
+		property test.name.skip.portal.instance = "SearchAPI#RequestKeywordsUsingDisplayLanguageId";
+
+		var parametersDisplay1 = "size=10&displayLanguageId=${displayLanguageId1}&page=0&groupId=${groupId}&sort=counts,desc,lastModifiedDate,desc";
+		var parametersDisplay2 = "size=10&displayLanguageId=${displayLanguageId2}&page=0&groupId=${groupId}&sort=counts,desc,lastModifiedDate,desc";
+		var displayLanguageId1 = "en-US";
+		var displayLanguageId2 = "pt-BR";
+		var groupId = JSONGroupSetter.setGroupId(groupName = "Guest");
```

Comment:
Hey @manuferreira, I think you don't need to use the `groupId` here. What do you think?